v21.1/movr-flask-use-case.md

```diff
@@ -0,0 +1,63 @@
+---
+title: MovR&#58; A Global Application Use-Case
+summary: This page covers an example use-case for global applications.
+toc: true
+redirect_from: multi-region-use-case.html
+---
+
+This page walks you through an example use-case for global application development and deployment. It is the first section of the [Develop and Deploy a Global Application](movr-flask-overview.html) tutorial.
+
+## Overview
+
+MovR is a fictional company that offers users a platform for sharing vehicles, like scooters, bicycles, and skateboards, in select cities across the United States and Europe. To serve users in two continents, the folks at MovR need an application that is globally available, [resilient to system failures](#resiliency-and-distributed-deployments), and [optimized for latency](#latency-in-global-applications).
+
+## Resiliency and distributed deployments
+
+For an application to be resilient to system failures, the application and database need to be deployed on multiple machines (i.e., part of a distributed deployment). In distributed CockroachDB deployments, all data is replicated and distributed across the instances of the database that make up the deployment. For more information about data replication and distribution in CockroachDB, see [The Raft Protocol in CockroachDB](https://www.youtube.com/watch?v=k5BR9m8o9ec&feature=youtu.be).
+
+The replication and distribution of data across multiple machines in a *single region* makes the deployment resilient to individual node failures within the region. Replication and distribution across *multiple regions* makes the deployment resilient to entire regional failures. To achieve the highest level of resiliency, we use a multi-region deployment for MovR database and application.
+
+{{site.data.alerts.callout_info}}
+In the [example deployment](movr-flask-deployment.html), the application and the database deployments are separate and not co-located on the same machine.
```

Comment:
So far this section talks about the benefits of the distribution of your CockroachDB nodes, and doesn't discuss whether to host the application on separate machines. Here, we're simply saying that in the example, the app and database are not 'on the same machine'; so that's a new concept that perhaps should be introduced prior to the note, but also, won't there be multiple DB machines, given the multi-region focus?

Not sure I'm explaining this well, but I'd consider additional detail on best practices above, and then more clarity in the note re. the architecture of the example and how it might differ from what you'd do in production (if that's the case).